src/state/account/auth.ts

```diff
@@ -59,4 +63,7 @@ export const refreshToken = async () => {
   Cookies.set('atoken', atoken, COOKIES_PARAMS);
 };
 
-export const getUserEmail = () => jwt(Cookies.get('atoken')).email;
+export const getUserEmail = () => {
+  const token = getToken();
+  return token === undefined ? undefined : jwt<AccessToken>(token).email;
```

Comment:
```suggestion
  return token ? jwt<AccessToken>(token).email : null;
```
Question: Is there a reason to use `undefined` instead of a falsy value in this case?

Reply:
answered above, just style preference